prow/kube/types.go

```diff
@@ -109,6 +109,8 @@ type Volume struct {
 	DownwardAPI *DownwardAPISource `json:"downwardAPI,omitempty"`
 	HostPath    *HostPathSource    `json:"hostPath,omitempty"`
 	ConfigMap   *ConfigMapSource   `json:"configMap,omitempty"`
+	// https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L954
```

Comment:
Linking to lines of code on master is always confusing. This one in particular is already wrong XD